library/modules/bigip_profile_client_ssl.py

```diff
@@ -666,6 +690,14 @@ def strict_resume(self):
             return 'yes'
         return 'no'
 
+    @property
+    def renegotiation(self):
+        if self._values['renegotiation'] is None:
```

Comment:
remove this this function already covers this:

https://github.com/F5Networks/f5-ansible/blob/devel/library/module_utils/network/f5/common.py#L222

just return result of flatten boolean